VERSION

```diff
@@ -1 +1 @@
-0.3.1-dev
\ No newline at end of file
+1.0.0
```

Comment:
Very nice that you bumped the major version, though, we remove the `-dev` tag only when we are doing a release.
```suggestion
1.0.0-dev
```

Reply:
:heavy_check_mark: